src/server/models/listItem/providers/Lawyers.ts

```diff
@@ -67,7 +63,7 @@ export async function findPublishedLawyersPerCountry(props: {
       offset,
     });
 
-    return await prisma.$queryRaw(query);
+    return await prisma.$queryRawUnsafe(query);
```

Comment:
Same question re: queryRawUnsafe

Reply:
I believe I've answered the question in [the comment above](https://github.com/UKForeignOffice/lists/pull/474#discussion_r981193421) but I'm happy to elaborate if you'd like.